src/dataMap/metaManager/mods/__tests__/dynamicCellMeta.unit.js

```diff
@@ -1,9 +1,9 @@
-import { DynamicCellMetaMod } from 'handsontable/dataMap/metaManager/mods/dynamicCellMeta';
-import MetaManager from 'handsontable/dataMap/metaManager';
-import Handsontable from 'handsontable';
-import Hooks from 'handsontable/pluginHooks';
+import { DynamicCellMetaMod } from '../dynamicCellMeta';
+import MetaManager from '../../';
+import Handsontable from '../../../../';
+import Hooks from '../../../../pluginHooks';
 
-jest.mock('handsontable');
+jest.mock('../../../../');
```

Comment:
```suggestion
jest.mock('handsontable');
```

Reply:
Can we revert the 3 and 6 lines? I think that with a relative path, the mock loses its readability.

Reply:
Of course! 🙂